### DISCUSSION TITLE
add sub-perms for /pweather

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#6940] [Enhancement]
Why not add sub-perms for the /pweather command? Right now, if you use /pweather rain, you can fly unlimited on a trident. And if this can't be fixed by checking the weather on the server, why not add sub-perms for each weather type? In that case you could set a negative permission for rain to block the /pweather rain and the problem would be solved

**Comment by mrfloris** [comment]
Visit https://github.com/Zrips/CMI/discussions/new to post new idea or suggestion
Posting new suggestion in ticket section could result in their removal.

When you create a new issue for new feature request, it says this in the text field, please post it there.
